main.c

```diff
@@ -900,14 +897,6 @@ int main(int argc, char **argv)
 	filecnt = fileidx;
 	fileidx = options->startnum < filecnt ? options->startnum : 0;
 
-	for (i = 0; i < ARRLEN(buttons); i++) {
-		if (buttons[i].cmd == i_cursor_navigate) {
-			imgcursor[0] = CURSOR_LEFT;
-			imgcursor[2] = CURSOR_RIGHT;
-			break;
-		}
-	}
-
```

Comment:
Wouldn't removing this cause issue?

Reply:
I just moved it into to config file so users had full control. Imagine a user wraps/extends i_cursor_navigate and use the new function for their mouse bindings? Would you expect your pointer icons to change?

Reply:
> I just moved it into to config file so users had full control.

Can you separate this into it's own commit then, we'll rebase the PR instead of squashing. I'd like to be able to have a rough idea of what a commit is changing just by reading the short commit msg on the commit log.

Reply:
Done